apps/emqx_durable_storage/test/emqx_ds_SUITE.erl

```diff
@@ -243,6 +243,245 @@ t_06_add_generation(_Config) ->
     end,
     lists:foldl(Checker, [], lists:zip(StartTimes, MsgsList)).
 
+t_08_smoke_list_drop_generation(_Config) ->
```

Comment:
It would be nice to have a comment explaining the purpose of the test at the high level; which property is it verifies.